src/app/layout/header/header.component.scss

```
@@ -24,6 +24,29 @@ li {
   max-width: calc(100vw - 2rem);
 }
 
+.compare_dropdown_panel {
+  width: max-content;
+  max-width: calc(100vw - 2rem);
+}
+
+.compare_dropdown_row.cdk-drag-preview {
+  @apply bg-secondary rounded-lg shadow border border-emerald-400;
+  box-sizing: border-box;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)